narwhals/_utils.py

```diff
@@ -603,11 +603,7 @@ def backend_version(implementation: Implementation, /) -> tuple[int, ...]:
     impl = implementation
     module_name = _IMPLEMENTATION_TO_MODULE_NAME.get(impl, impl.value)
     native_namespace = _import_native_namespace(module_name)
-    if impl.is_polars():
-        from importlib import metadata
-
-        into_version = metadata.version("polars")
```

Comment:
@MarcoGorelli the issue here is that v1.32.0 will be mapped to 0.50.0 or something like that. Hence all the `backend_version < (1, )` will be evaluated to true

Reply:
1.32.0 is fine, it's 1.32.1 that was problematic, and that one's been yanked from pypi